src/map/script.hpp

```diff
@@ -1895,6 +1895,43 @@ enum e_hat_effects {
 	HAT_EF_SUBJECT_AURA_WHITE,
 	HAT_EF_SUBJECT_AURA_RED,
 	HAT_EF_C_SHINING_ANGEL_WING,
+	HAT_EF_MAGIC_STAR_TW,
+	HAT_DIGITAL_SPACE,
```

Comment:
I know this is a copy from kRO, but do you mind adjusting this to `HAT_EF_DIGITAL_SPACE`. Don't forget to update the export and item as well.

Reply:
sure i will fix later